src/test/java/org/folio/rest/impl/UserImportAPITest.java

```diff
@@ -300,23 +300,20 @@ public void testImportWithUserWithoutExternalSystemId() throws IOException {
       .statusCode(422);
   }
 
-  /*
-   * This test does not reflect real-time environment currently.
-   */
-  //  @Test
-  public void testImportWithUserWithEmptyExternalSystemId() throws IOException {
+  @Test
+  public void testImportWithUserWithMaskedExternalSystemId() throws IOException {
 
-    mock.setMockJsonContent("mock_user_creation_with_empty_externalsystemid.json");
+    mock.setMockJsonContent("mock_user_creation_with_masked_externalsystemid.json");
 
     List<User> users = new ArrayList<>();
     User testUser = generateUser("1234567", "Amy", "Cabble", null);
-    testUser.setExternalSystemId("");
+    testUser.setExternalSystemId("amy/cab*ble");
```

Comment:
Should there be perhaps additional tests with additional special characters besides the forward slash?